test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/FhirOperationDataStoreTests.cs

```diff
@@ -18,23 +18,25 @@
 namespace Microsoft.Health.Fhir.Tests.Integration.Features.Operations
 {
     [Collection(FhirOperationTestConstants.FhirOperationTests)]
-    [FhirStorageTestsFixtureArgumentSets(DataStore.CosmosDb)]
-    public class CosmosFhirOperationDataStoreTests : IClassFixture<FhirStorageTestsFixture>, IAsyncLifetime
+    [FhirStorageTestsFixtureArgumentSets(DataStore.All)]
+    public class FhirOperationDataStoreTests : IClassFixture<FhirStorageTestsFixture>, IAsyncLifetime
     {
-        private IFhirOperationDataStore _operationDataStore;
-        private IFhirStorageTestHelper _testHelper;
+        private readonly IFhirOperationDataStore _operationDataStore;
+        private readonly IFhirStorageTestHelper _testHelper;
+        private readonly CancellationToken _cancellationToken;
 
         private readonly CreateExportRequest _exportRequest = new CreateExportRequest(new Uri("http://localhost/ExportJob"), "destinationType", "destinationConnection");
 
-        public CosmosFhirOperationDataStoreTests(FhirStorageTestsFixture fixture)
+        public FhirOperationDataStoreTests(FhirStorageTestsFixture fixture)
         {
             _operationDataStore = fixture.OperationDataStore;
             _testHelper = fixture.TestHelper;
+            _cancellationToken = new CancellationTokenSource().Token;
```

Comment:
We would probably be better off storing the `CancellationTokenSource` in a field so that we could call `CancellationTokenSource.Cancel()`. 

In looking at the code, we may just want to update the `DeleteAllExportJobRecordsAsync` method to have an optional cancellation token parameter (e.g. `CancellationToken cancellationToken = default` and not actually pass it in from here since we're never actually cancelling the task.

Reply:
Sounds good, made the cancellation token an optional parameter in 0cabf1a.